src/test/java/emissary/core/BaseDataObjectTest.java

```diff
@@ -1406,4 +1409,34 @@ void testArrayInArrayOutNoSet() throws IOException {
             logbackTester.checkLogList(LEVELS_ONE_WARN, ONE_UNSAFE_MODIFICATION_DETECTED, NO_THROWABLES);
         }
     }
+
+    @Test
+    void testNewInputStream() throws IOException {
```

Comment:
The variable names in this method hurt my brain.  In the span of 14 actual lines of code, the word "byte" is repeated _26 times_, and it's just too much cognitive noise.